src/redux/orders/orderThunk.ts

```diff
@@ -0,0 +1,11 @@
+import Service from 'src/services/services'
+import { ordersLoadStart, ordersLoadFinish } from '../reducers/ordersSlice'
+import { AppDispatch } from '../store'
+
+export const loadOrdersAsync = () => (dispatch: AppDispatch) => {
+  dispatch(ordersLoadStart())
+
+  Service.getAllOrders().then((response) =>
+    dispatch(ordersLoadFinish(response.data)),
+  )
+}
```

Comment:
Created a wrapper for all requests. Why create a separate file with a request. If you can use a wrapper